### PR TITLE
Resolve Login Issues

### PR DESCRIPTION
**Pull request by ahinton-league**
fixed package id issue by changing the value in the google-play.json
also resolved issue where genesis wasn't initialized
the login sometimes errors out with this error message, but it eventually works:
```
java.lang.ClassNotFoundException: com.auth0.android.provider.CustomTabsOptions
        at java.lang.Class.classForName(Native Method)
        at java.lang.Class.forName(Class.java:454)
        at android.os.Parcel.readParcelableCreator(Parcel.java:3031)
```

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -91,6 +91,9 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
 dependencies {
     implementation project(":tcn-client-android:tcn-client-android")
 
+    // I think we need this to prevent a random unmarshalling error with login
+    implementation 'com.auth0.android:auth0:2.5.0'
```
Comment (ahinton-league) [inline app/build.gradle:95]: This actually didn't fix it :(

File: app/src/main/java/org/covidwatch/android/data/signedreport/firestore/SignedReportsDownloadWorker.kt [hunk 2]
```diff
@@ -30,79 +30,79 @@ class SignedReportsDownloadWorker(var context: Context, workerParams: WorkerPara
...
+//                            }
+//                        } else {
+//                            result = Result.failure()
+//                        }
+//                        null
+//                    })
+//
+//        Tasks.await(task)
+//
+//        Log.i(TAG, "Finish task")
```
Comment (ahinton-league) [inline app/src/main/java/org/covidwatch/android/data/signedreport/firestore/SignedReportsDownloadWorker.kt:103]: The app was crashing here due to a firebase error, so I commented it out for now

Comment (m-nogas) [reply app/src/main/java/org/covidwatch/android/data/signedreport/firestore/SignedReportsDownloadWorker.kt:103]: Ahh could have been introduced when I migrated the project off of deprecated crashlytics using fabric

**Review by m-nogas — APPROVED** [review]
Thank you!

**Pull request merged**